### PR TITLE
Use pinned version of the operations repository

### PR DESCRIPTION
**Pull request by aquarat**
This makes services use a tagged version of the operations repository - the repository state that it currently uses, so that when we update the operations repository it won't break anything here.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -33,17 +33,23 @@ export function getServiceData(apiName: string, beaconName: string, chain: strin
   const goBeaconData = goSync(() =>
     require(`@api3dao/operations/data/apis/${apiName}/beacons/${sanitiseFilename(beaconName)}.json`)
   );
-  if (!isGoSuccess(goBeaconData)) throw new Error(`Service data does not exist.`);
+  if (!isGoSuccess(goBeaconData)) throw new Error(`Beacon service data does not exist.`);
   const beaconData = goBeaconData.data;
 
+  const goTemplateData = goSync(() =>
```
Comment (Ashar2shahid) [inline src/index.ts:39]: I just noticed that this was referring the beacon parameters and not the template parameters. It should have been renamed better initially, but I guess since we won't be using services atm, we can let it be and come back and refactor it later

**Review by Ashar2shahid — APPROVED** [review]
LGTM 👍

File: src/index.ts (hunk 1 above)
Comment (aquarat) [reply src/index.ts:39]: Hey, yeah, I didn't create this repo, but it's good that you noticed this :)

AFAIK we won't use services again (ops will be the new reference for beacons v0.2), so this is just a caretaker action essentially, as you noticed. Thanks for reviewing :)

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -26,7 +26,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '14'
-          cache: 'yarn'
```
Comment (Siegrift) [inline .github/workflows/main.yml:29]: Why did you remove this?

Comment (Ashar2shahid) [reply .github/workflows/main.yml:29]: I'm guessing because ops repository still has v1.0.0 and it was still referring to that and not pulling the new changes.
